desc/io/ascii_io.py

```diff
@@ -1,11 +1,21 @@
 """Legacy functions for reading and writing ascii format."""
 
+import warnings
+
 import numpy as np
 
+from desc.utils import errorif
+
 
-def write_ascii(fname, eq):
+def write_ascii(fname, eq):  # noqa: C901
     """Print the equilibrium solution to a text file.
 
+    Note: will save the pressure and iota profile as
+    power series profiles. If not an iota constrained
+    equilibrium, the iota profile will be computed and
+    fit with a fourier series of the equilibrium's L
```

Comment:
```suggestion
    fit with a power series of the equilibrium's L
```
I assume this is a typo?